### ISSUE TITLE
Ghostery uses Bunny CDN

### ISSUE THREAD

**Issue by Vinalti** [ghostery/ghostery-extension#1031]
### Description

Ghostery extension is constantly sending requests to malicious IP addresses belonging to `*.bunnyinfra.net`   

### Expected Behavior

When browser is not open (but in background) or no website is open, ghostery should not be sending unsolicited requests.

### Actual Behavior

Network connections are initiated, about once every minute, to differents IPs on port 443. The traffic is detected as malicious and blocked by an external router.  
Even when Browser is closed (if "keep background app open when browser is closed" option is enabled) or inactive the requests keep being sent.

All reached IPs belongs to the `*.bunnyinfra.net` network. 
Example of IPS:
```
 89.187.169.39
 89.187.169.47
138.199.36.7
138.199.36.11
138.199.37.229
138.199.37.231
169.150.247.37
169.150.247.33
185.59.220.194
185.59.220.198
```

### Steps to Reproduce

1. Observe connections attempts on your computer (powershell: Get-NetTCPConnections, or Ressource Monitor)
2. Start your browser (with ghostery installed)
3. Repeat step 1, search for IP belonging in above mentioned IP Range. (`PID` / `OwningProcess` should be `0` or `- `)

I cannot affirm that those connections are occuring once a minute only because they are blocked or if they would nevertheless occur once a minute even when successful.

### Versions

+ Browser: Chrome / Edge (several)
+ OS: Windows 10 (several)
+ Ghostery (last public version as of 10/02/2023)

### Classification
I would consider this a security issue: either the application has been compromised and is trying to extract data from the browser and send it to a malicious server, or the hosting servers that Ghostery uses are considered malicious and that prevent proper detection of truly malicious activity. In the later case, I would advise changing provider.   
On my side I removed the extension from all my devices.

**Comment by chrmod** [comment]
Dear @Vinalti, thank you for opening this issue. Security and privacy of all our users is our top priority. 

TLDR; Ghostery uses Bunny CDN to distribute ad-blocking block lists. This is perfectly expected situation and there is no security/privacy threat. 

None of extension initiated requests should use direct IPs. That said, we are using [Bunny CDN](https://bunny.net/) to deliver our ad-blocking block lists, anti-tracking token list and other resources required for extension to work. 

When testing locally:
https://cdn.ghostery.com/adblocker/configs/desktop-full/allowed-lists.json
resolves to `169.150.247.40` which is in the same IP range you've listed in the issue description. 

If you don't want to receive automatic updates, you can always disable them in the settings:
<img width="292" alt="Screenshot 2023-02-14 at 12 18 02" src="https://user-images.githubusercontent.com/1228153/218721886-e3ed9a1c-7144-44bf-b2bf-a1fb4ba52dbf.png">

From your description it looks to like the content blocking software on your router for some reason decided that Bunny CDN should be blocked. Decision to block whole IP range of general purpose CDN is likely resulting in many websites and applications not working as expected. 

We would recommend opening an issue for your router's content blocker software, as blocking CDNs is probably not desirable for majority of users.

**Comment by Vinalti** [comment]
Dear @chrmod,

Thank you very much for your detailed answer.
I believe there must have been malicious websites using this CDN due to its low price, and because of that it was marked as malicious. 
From what I remember this router is using a Google provided list of malicious IPs / Domains. Therefore, I would tend to think that I am not the only one in this situation.
I didn't notice issues on other websites so far.

**Comment by chrmod** [comment]
If you could name the software, we can forward the issue to Bunny CDN, in the end it is in interest of all parties to get this situation straight. 

Thanks to your report, we have also realized that our update mechanism is a bit too eager and we should not retry to reconnect so often. We will incorporate this knowledge in the upcoming Ghostery for Manifest V3. 

At Ghostery we value contributions! Please reach out to support@ghostery.com with a link to this issue, so we can send you a coupon code to get access to Contributor extras.
We are happy to have you as a part of the Ghostery Contributors community!

**Comment by chrmod** [comment]
closing for now
